client/errors.go

```diff
@@ -0,0 +1,130 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package client
+
+// XXX: sync with the error kinds in daemon and ensure we define them
+//      only in a single place
+
+// Keep in sync with: https://forum.snapcraft.io/t/17954
+const (
+	// ErrorKindTwoFactorRequired the client needs to retry the
```

Comment:
If there's a convention to use a colon after the error name then this line should be adjusted.
```suggestion
	// ErrorKindTwoFactorRequired: the client needs to retry the
```